frontend/src/routes/data/CustomData/custom-fields/customComponents/SelectDropdownCustom.tsx

```
@@ -0,0 +1,136 @@
+                    }
+                  : itemIter,
+              ),
+            }
+          : field,
+      );
+    });
+  };
+  return (
+    <Field>
+      <Label className="block text-sm font-medium leading-6 mb-2 text-gray-300">
+        {title + ' '}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)